backend/danswer/access/document_set_utils.py

```diff
@@ -0,0 +1,19 @@
+from uuid import UUID
+
+from danswer.configs.constants import PUBLIC_DOCUMENT_SET
+
+
+def build_personal_document_set_name(user_id: UUID | None) -> str:
+    if user_id is None:
+        return PUBLIC_DOCUMENT_SET
+
+    return f"__USER_{user_id}"
```

Comment:
nit: prefer having this be PERSONAL_DOCUMENT_SET.format(user_id) or something for consistency